frontend/src/components/Search/SearchForm/SearchForm.tsx

```diff
@@ -252,18 +237,21 @@ export const Search: React.FC<{
 	const cursorIndex = inputRef.current?.selectionStart || 0
 	const activeTermIndex = getActiveTermIndex(cursorIndex, queryTerms)
 	const activeTerm = queryTerms[activeTermIndex]
+	const debouncedKeyValue = useDebouncedValue<string>(activeTerm.value)
 
+	// TODO: code smell, user is not able to use "message" as a search key
+	// because we are reserving it for the body implicitly
```

Comment:
hmm, is there some cleaner thing we can do about this?

Reply:
Went back and forth on this with @ccschmitz - I think the proposed solution is outlined in [this slack message](https://highlightcorp.slack.com/archives/C0454PWLLLW/p1699401258239049?thread_ts=1699390242.808739&cid=C0454PWLLLW)